website/docs/compute/get-started.md

```diff
@@ -0,0 +1,205 @@
+
+## Generate your token
+You can get a Compute Environment token either with our [environment token generator web app](https://suborbital.network/) or with the `subo` command line tool.
+
+:::caution  
+The domain of your email address should match the `environment` you 
+configure for Compute; i.e., if your email is `laika@example.com`, your 
+Compute environment would be called `com.example`. See [Fully-qualified 
+function names](./customizing-functions/fully-qualified-function-names.md) 
+for more 
+information.
+:::
+
+To create your token, run:
+
+```bash
+subo compute create token <email>
+```
+
+A verification code will be sent to your email address, and the token will be 
+used to authenticate you and link your Compute installation to you.
+
+:::tip
+Subo will print out your token in your terminal, and cache it for use in 
+subsequent steps.
+:::
+
+Keep your environment token safe (store it in your password manager), and 
+only use it when running or installing Compute. If you lose it, just repeat 
+this token generation process.
+
+
+## Run Compute locally
+
+To run Compute locally, you'll use the Subo CLI and [Docker](https://docs.docker.com), both of which must be installed before continuing.
+
+### Prepare your local environment
+
+The Subo CLI will create some files on disk to set up your local deployment, so get started by creating a `suborbital` directory somewhere on your filesystem:
+
+```bash
+mkdir suborbital
+cd suborbital
+```
+
+### Start Compute
+
+Next, use Subo to start your local Compute Core instance:
+
+```bash
+subo compute deploy core --local
+```
+
+You may be asked to enter your Environment Token, and then Subo will use `docker-compose` to launch your Compute Core instance automatically. Compute runs in the background by default. You can use `docker-compose logs -f` to view the logs of the running containers. Run `docker-compose down` to terminate the containers.
+
+### Run Compute
+When you run Compute, it will wait for you to press enter to start a REPL 
+where you can add or edit functions (see [Meet the Compute Editor](get-started#meet-the-compute-editor)).
+
+```
+✅ DONE: ready to start installation
+⏩ START: installing...
+▶️ docker-compose up -d
+Container suborbital-scc-control-plane-1  Recreate
+Container suborbital-scc-control-plane-1  Recreated
+Container suborbital-scc-atmo-1  Recreate
+Container suborbital-scc-atmo-1  Recreated
+Container suborbital-scc-control-plane-1  Starting
+Container suborbital-scc-builder-1  Starting
+Container suborbital-scc-control-plane-1  Started
+Container suborbital-scc-builder-1  Started
+Container suborbital-scc-atmo-1  Starting
+Container suborbital-scc-atmo-1  Started
+ℹ️  use `docker ps` and `docker-compose logs` to check deployment status
+
+PROXY: local tunnel to function editor started
+
+
+Press enter to launch the local Compute REPL...
+
+
+1. Create or edit a function
+
+Choose an option: 1
+
+To create or edit a function, enter its name (or FQFN): new-fn
```

Comment:
Can we keep the function name consistent across the docs please?
On the function editor image above we use `hello`, so let's stick with that?